handlers/isClientExist.js

```diff
@@ -0,0 +1,12 @@
+const isClientExist = async (Model, req) => {
+    // check for existing employee
+    const { company, managerSurname, managerName, email, phone } = req.body
+    let isExist;
+    if (company || managerSurname || managerName || email || phone) {
+        isExist = await Model.findOne({ company, managerName, managerSurname, email, phone })
```

Comment:
please verify only email & phone

Reply:
<img width="1132" alt="image" src="https://github.com/idurar/idurar-erp-crm/assets/96794196/018fd6f4-47f3-4e14-8df9-71010a969263">
@salahlalami can you check now

Reply:
@OvilashMoitra remove unused variable , keep only phone & email

Reply:
@OvilashMoitra check also if company name exist ,
if exist end  return id of client

Reply:
<img width="1005" alt="image" src="https://github.com/idurar/idurar-erp-crm/assets/96794196/93f93d2a-8b79-4378-ba5d-39d7df6c0ffc">
<img width="737" alt="image" src="https://github.com/idurar/idurar-erp-crm/assets/96794196/41af2ece-0c52-434f-8018-6c859d35d32b">
@salahlalami Can you check now.